README.md

```diff
@@ -89,8 +89,8 @@ To see a list of your current sites you can use `ddev list`.
 ```
 ➜  ddev list
 1 local site found.
-NAME   	TYPE   	URL                      	DATABASE URL   	STATUS
-drupal8	drupal8	http://drupal8.ddev.local	127.0.0.1:32852	running
+NAME   	TYPE   	LOCATION                        URL                      	DATABASE URL
+drupal8	drupal8	/Users/username/drupal8/docroot	http://drupal8.ddev.local	drupal8.ddev.local:3306
```

Comment:
We discussed in #153 shortening homedir if we detect it to help keep the width short. It's not critical by any means, but it would be nice to have.